modules/ROOT/pages/afm-upload-monitor.adoc

```diff
@@ -21,33 +21,28 @@ You can upload a suite of tests that you have written and want to run as a monit
 == Procedure
 To create a monitor by uploading a test suite:
 
-. In the *Functional Monitoring* section of Anypoint Monitoring, click *Upload*.
-. Upload the folder that contains your test suite.
-. Fill in the fields with the information that you have gathered.
-. Click *Upload*.
+. In the *Functional Monitoring* section of Anypoint Monitoring, click *Upload Monitor*.
+. Select the folder that contains your test suite.
++
+For details on file structure and file requirements, see xref:bat-example-test-suite.adoc[Example of a Test Suite].
 
-== Result
+. Click *Upload Monitor*.
+. Click *Next*.
 
-The dashboard opens. The monitor is listed on the left. Beneath the name of the monitor is an indicator that explains when the current run of the monitor began. A line at the bottom of the list item shows how much time remains before the next run of the monitor. Also, the *Monitored Endpoints*, *Tests*, and *Schedules* sections will all be empty. After the monitor has run, the list item for the monitor will indicate whether the monitored endpoint passed or failed.
+== Result
 
+The monitor detail opens. An indicator displays how long the execution ran. The last executions list displays the test result.
```

Comment:
What is "the monitor detail"? Does this have a UI label? Is it "the monitor's detail screen"?

Reply:
Again with "an indicator": what is this, exactly?